specs/AllowGoBackForward.md

```diff
@@ -0,0 +1,142 @@
+# Background
+Some developers may want to disable navigating back or forward to make a single page displayed, and not allow the users to navigate away from it accidentally. So we can disallow navigating back and forward by setting the API.
+
+# Description
+We add new `AllowGoBack` and new `AllowGoForward` properties in `CoreWebView2Settings`.
+These two APIs allow end developers to toggle the allow go back and forward functionality easily.
+If we set these APIs to disabled, users can not navigate back and forward regardless of source (button click, xbutton click, key event and etc.). The `CanGoBack` and `CanGoForward` APIs will also return false if we set `AllowGoBack` and `AllowGoForward` to disabled.
```

Comment:
It would be good to describe how this relates to NavigationStarting. Something like:
```suggestion
Although the CoreWebView2.NavigationStarting event can be used to cancel navigations it is not easy to distinguish between back and forward navigations from other sources of navigation.
If we set these APIs to disabled, users can not navigate back and forward regardless of source (button click, xbutton click, key event and etc.). The `CanGoBack` and `CanGoForward` APIs will also return false if we set `AllowGoBack` and `AllowGoForward` to disabled.
```

Reply:
Done, thanks.